low-level/files/opt/seagate/sspl/conf/sspl.conf.LR2.yaml

```diff
@@ -165,7 +165,8 @@ NODEHWSENSOR:
 
 REALSTORLOGICALVOLUMESENSOR:
    threaded: true
-   polling_frequency_override: 0
+   # The 'polling_frequency_override' is getting used by both disk-group & volume sensor.
```

Comment:
# Polling frequency override here applicable for both Disk Group & Logical Volume monitoring